tests/Cms/System/SystemTest.php

```diff
@@ -339,6 +339,16 @@ public function testInitPermission($root)
 		new System($app);
 	}
 
+	/**
+	 * @covers ::info
+	 */
+	public function testIinfo()
+	{
+		$app = $this->app->clone([]);
+		$system = new System($app);
+		$this->assertIsArray($system->info());
```

Comment:
With type hinting, the only thing this assertion does is to check whether the method explodes if it's being called. For that I'd rather use `$system->info(); $this->expectNotToPerformAssertions();`. But the usefulness of the test is still almost zero.

I get that the Kirby and PHP versions cannot be tested well and the server software and license label are unlikely to break, but maybe we could at least test the languages array as it is built using actual logic.